pro/cypress/e2e/step-definitions/searchIndividualOffer.cy.ts

```diff
@@ -15,3 +20,39 @@ When('I select offerer {string} in offer page', (offererName: string) => {
   cy.wait('@getOffererId')
   cy.findAllByTestId('spinner').should('not.exist')
 })
+
+When('I select {string} in offer status', (filter: string) => {
+  cy.findByTestId('wrapper-status').within(() => {
+    cy.get('select').select(filter)
+  })
+})
+
+When('I select a date in one month', () => {
+  const dateSearch = dayjs().add(30, 'days').format('YYYY-MM-DD')
```

Comment:
Est-ce qu'il y a vraiment besoin de rajouter une lib uniquement pour ça ?

Reply:
que suggères-tu pour que la date du jour + 30 jours au format `yyyy-mm-dd` soit toujours correcte quel que soit le jour de l'année, la timezone, etc ?

Reply:
Complètement d'accord avec Amine, on a déjà `date-fns` qui devrait suffire (où même l'api Date ?).

@scolson-pass @ncanseco-pass Pour info côté frontend on a pour pratique de minimiser le nombre de librairies qu'on installe. 
Chaque librairie c'est de la maintenance en plus, une augmentation de la taille du code téléchargé par les clients, et moins d'homogénéité dans les pratiques de développement.
Par ailleurs, si une librairie ne sert pas dans l'app en production, il vaut mieux mettre le package dans les `devDependencies`.

Reply:
OK, vous avez tout à fait raison, j'étais passé à côté de l'utilisation déjà existante de `data-fns` que je ne connaissais pas. C'est corrigé/commité. Merci pour les commentaires